internal/controllers/machine/machine_controller_status.go

```diff
@@ -659,7 +659,7 @@ func setAvailableCondition(_ context.Context, machine *clusterv1.Machine) {
 		v1beta2conditions.Set(machine, metav1.Condition{
```

Comment:
What about adding some kind of log like we have it for the ready condition?

If we don't log any error "Please check controller logs for errors" is pretty annoying :)